src/features/emails/components/supportEmailTemplate.tsx

```
@@ -0,0 +1,33 @@
+  description,
+}: TemplateProps) => {
+  return (
+    <div style={styles.container}>
+      <p style={styles.greetings}>New Support Request</p>
+      <p style={styles.textWithMargin}>
+        From: {from} <br />
+        Subject: {subject} <br />
+        Message:
+      </p>
+      <p
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)